scripts/barclamp_mgmt_lib.rb

```diff
@@ -114,18 +93,6 @@ def catalog(bc_path)
     cat['barclamps'][name] = {} if cat['barclamps'][name].nil?
     description = bc['barclamp']['description']
     display = bc['barclamp']['display']
-    if description.nil?
-      debug "Trying to find description"
-      [ File.join(bc_path, '..', name, 'chef', 'data_bags', 'crowbar', "bc-template-#{name}.json"), \
-        File.join(bc_path, '..', "barclamp-#{name}", 'chef', 'data_bags', 'crowbar', "bc-template-#{name}.json"), \
-        File.join(bc_path, '..', '..', 'chef', 'data_bags', 'crowbar', "bc-template-#{name}.json") ].each do |f|
-        next unless File.exist? f
-        s = JSON::load File.open(f, 'r')
-        description = s['description'] unless s.nil?
-        break if description
-      end
-    end
-    # template = File.join bc_path, name,
```

Comment:
I'm fine dropping that, but we first need to fix the yml files that don't have a description. I could find nfs_client, ntp and suse-manger-client just in crowbar-core. I would also add a big nice warning if it's nil.